app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/SpeakFragment.java

```diff
@@ -143,6 +147,8 @@ public void onClick(View v)
         });
         helloReddit.setTypeface(tf);
 
+        spokenString = "why are my hands so smelly?";
+        new GetWordsAsync(spokenString, importantWords, getActivity(), commentText).execute();
```

Comment:
could we integrate with the voice recognition rather than using a hardcoded string?